src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -183,16 +183,17 @@ export function getIconClassNameForBeacon(authenticatorKey) {
   return getAuthenticatorData({ authenticatorKey }).iconClassName;
 }
 
-export function removeRequirementsFromError(errorJSON, policy) {
-  const passwordRequirementsAsString = getPasswordComplexityDescription(policy);
+export function removeRequirementsFromError(errorJSON) {
+  const passwordRequirementsNotMet = loc('registration.error.password.passwordRequirementsNotMet', 'login');
   if (errorJSON.errorCauses?.length > 0
     && Array.isArray(errorJSON.errorCauses[0].errorSummary)
     && errorJSON.errorCauses[0].errorSummary.length > 0) {
-
-    // Remove the requirements string if it is present.
-    errorJSON.errorCauses[0].errorSummary = errorJSON.errorCauses[0].errorSummary[0]
-      .replace(`${passwordRequirementsAsString}`, '')
-      .trim();
+    // Change from Array to string for all errors.
+    errorJSON.errorCauses[0].errorSummary = errorJSON.errorCauses[0].errorSummary[0];
```

Comment:
Is is possible to check an error code instead of this localized string ?

Reply:
I think you were talking about line 196 below. I have updated it now.